### ISSUE TITLE
arrow allignment is not detecting misalignment in selector or hash.

### ISSUE THREAD

**Issue by Heidistein** [puppetlabs/puppet-lint#226]
## Describe the Bug
Also noted in https://github.com/rodjek/puppet-lint/issues/951
Considering the following hash creation:
```puppet
  $_value = $value ? {
    Array => $value.flatten().join(','),
    true => 'yes',
    false => 'no',
    default => $value,
  }
```
there is no raised warning about the obvious alignment failure.

## Expected Behavior
A warning or error.

**Comment by gerases** [comment]
Yes, the ruby code for the puppet-lint arrow-alignment plugin is a big loop around resources and not anything else. It starts like this:

```
COMMENT_TYPES = Set[:COMMENT, :SLASH_COMMENT, :MLCOMMENT]

PuppetLint.new_check(:my_arrow_alignment) do
  def check
    resource_indexes.each do |res_idx|
       ...
    end
      ...     
  end
```
